README.md

```diff
@@ -6,15 +6,19 @@
 
 A challenging optimization problem in the airline industry is determining which
 airports should be hub locations for an airline. In this demo, we show how to
-formulate this problem as a discrete quadratic model and use a hybrid solver to
-optimize and find feasible solutions.
+formulate this problem as either a discrete or constrained quadratic model and
+use a hybrid solver to optimize and find feasible solutions.
```

Comment:
Since "CQM" and "DQM" are used below, suggest defining those terms here:
```suggestion
formulate this problem as either a Discrete Quadratic Model (DQM) or Constrained
Quadratic Model (DQM) and use a hybrid solver to optimize and find feasible
solutions.
```